pkg/operator/webhookdeployment/webhook.go

```diff
@@ -151,8 +151,7 @@ func (c *csiSnapshotWebhookController) sync(ctx context.Context, syncCtx factory
 	if err != nil {
 		return err
 	}
-	lastWebhookGeneration := resourcemerge.ExpectedValidatingWebhooksConfiguration(webhookConfig.Name, opStatus.Generations)
-	webhookConfig, _, err = resourceapply.ApplyValidatingWebhookConfiguration(ctx, c.kubeClient.AdmissionregistrationV1(), syncCtx.Recorder(), webhookConfig, lastWebhookGeneration)
```

Comment:
I don't have the full context on this, any advice would be appreciated. The `lastWebhookGeneration` was removed from this method in the latest library-go in this [pr](https://github.com/openshift/library-go/pull/1256). The unit test will break because this behavior seemed to be desired. I could replicate the logic as it was before with some extra calls but would appreciate any guidance.

Reply:
/cc @bertinatto

Reply:
Thanks! Updated